### PR TITLE
[#ICC-115] Add EVH for message view Payment updates

### PR DESCRIPTION
**Pull request by AleDore**
#### List of Changes

- Add `UpdatePaymentOnMessageView` function that reads from a Kafka topic and update message view with related payment update information
- Add `HandlePaymentUpdateFailures` function that handles previous function failure in case backoff policy reach maximum retries
- Add unit tests for both functions
- Refactor some code
- Add env variables
#### Motivation and Context

This PR is necessary in order to update Message View with payment update informations from Payment Updater component.
Please note that this integration must be released after we update our infrastructure configuration.
#### How Has This Been Tested?

Unit Test
#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: HandlePaymentUpdateFailures/function.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      "type": "queueTrigger",
+      "direction": "in",
+      "name": "messageviewfailure",
+      "queueName": "%MESSAGE_VIEW_PAYMENT_UPDATE_FAILURE_QUEUE_NAME%",
+      "connection":"INTERNAL_STORAGE_CONNECTION_STRING"
+    }
+  ],
+  "retry": {
+    "strategy": "exponentialBackoff",
+    "maxRetryCount": 5,
```
Comment (fabriziopapi) [inline HandlePaymentUpdateFailures/function.json:13]: NB:
Queue triggers already have a default hidden retry set to 5.
With the function retry se to 5 too, this function will be retried for a max of 25 times.

File: UpdatePaymentOnMessageView/handler.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+          isSuccess: "false",
+          modelId: err.modelId ?? "",
+          name: "message.view.paymentupdate.failure"
+        },
+        tagOverrides: { samplingEnabled: String(isTransient) }
+      });
+      context.log.error(error);
+      if (
+        isTransient &&
+        context.executionContext.retryContext?.retryCount !==
```
Comment (fabriziopapi) [inline UpdatePaymentOnMessageView/handler.ts:55]: with the !== condition, at the last retry the function will try to publish on storage queue only once.
Could be safer to set the function retry to 15 and the condition to 
< context.executionContext.retryContext?.maxRetryCount - 5
?

File: HandlePaymentUpdateFailures/function.json (hunk 1 above)
Comment (AleDore) [reply HandlePaymentUpdateFailures/function.json:13]: Ok so we can remove this config ?

Comment (fabriziopapi) [reply HandlePaymentUpdateFailures/function.json:13]: if 5 retry are enough, we can remote it.

**Review by fabriziopapi — APPROVED** [review]

**Pull request merged**